src/Microsoft.VisualStudio.ProjectSystem.Managed/Collections/IConcurrentHashSet.cs

```diff
@@ -47,5 +47,10 @@ internal interface IConcurrentHashSet<T> : IEnumerable<T>
         /// otherwise, <c>false</c>
         /// </returns>
         public bool AddRange(IEnumerable<T> elements);
+
+        /// <summary>
+        /// Removes all elements from System.Collections.Concurrent.IConcurrentHashSet`1
+        /// </summary>
+        public void Clear();
```

Comment:
This is an interface, so we should not have visibility modifiers. I know it's not related to this PR, but could you also remove them from this and the other members in this interface?

```suggestion
        void Clear();
```